sections/how-we-impact.liquid

```diff
@@ -0,0 +1,163 @@
+{{ 'section-how-we-impact.css' | asset_url | stylesheet_tag }}
+
+{%- style -%}
+  .how_we_impact_bg_container {
+    margin-top: {{ section.settings.margin_top | times: 0.75 | round: 0 }}px;
+    margin-bottom: {{ section.settings.margin_bottom | times: 0.75 | round: 0 }}px;
+  }
+  @media screen and (min-width: 750px) {
+    .how_we_impact_bg_container {
```

Comment:
avoid these kind of class names, use descriptive names that refer to section's style or structure instead of the content.